test/external-modules/jvm-crash/src/main/java/org/elasticsearch/test/jvm_crash/RestJvmCrashAction.java

```diff
@@ -30,7 +30,8 @@ public class RestJvmCrashAction implements RestHandler {
     static {
         try {
             AccessController.doPrivileged((PrivilegedExceptionAction<?>) () -> {
-                Class<?> unsafe = Class.forName("sun.misc.Unsafe");
+                // Bypass UberModuleClassLoader, as server does not have our very dangerous permissions
```

Comment:
This shows an interesting side effect of using UberModuleClassLoader: for class loading, the server module/ProtectionDomain gets in the way. So if it does not have permissions (like `accessClassInPackage.sun.misc`), no matter if we grant them to the plugin, it won't work. But I don't think this is a problem.